src/components/Picker/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+
+const styles = theme => ({
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  formControl: {
+    margin: theme.spacing.unit,
+    minWidth: 120,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing.unit * 2,
+  },
+});
+
+const Picker = ({
+  classes, identifier, onSelection, options, selectedValue,
+}) => (
+  <form className={classes.root} autoComplete="off">
+    <FormControl className={classes.formControl}>
+      <Select
+        value={selectedValue}
+        onChange={onSelection}
+        inputProps={{ name: identifier, id: identifier }}
+      >
+        <MenuItem value={selectedValue}><em>{selectedValue}</em></MenuItem>
```

Comment:
No need for the first `MenuItem`. A simple dropdown should not have two items selected.

![screen shot 2018-07-26 at 11 15 36 am](https://user-images.githubusercontent.com/3766511/43271424-76ffa7d8-90c5-11e8-851a-03a7e57ed8df.png)